makefile

```diff
@@ -0,0 +1,20 @@
+GO=$(shell which go)
```

Comment:
nit, the first letter of the filename should be capitalized

Reply:
Removed. Not needed.